### PR TITLE
Approx tests

### PR DESCRIPTION
**Pull request by mgeplf**

### CODE REVIEW

File: tests/test_morphology_readers.cpp [hunk 1]
```diff
@@ -157,8 +147,7 @@ TEST_CASE("LoadH5MorphologySingleNeurite", "[morphology]") {
         const morphio::Morphology m("data/h5/v1/single-neurite.h5");
         REQUIRE(m.soma().points().empty());
         REQUIRE(m.points().size() == 3);
-        REQUIRE(almost_equal(m.points()[0][0], 4., 0.001));
-        REQUIRE(almost_equal(0., 0., 0.1));
```
Comment (mgeplf) [inline tests/test_morphology_readers.cpp:161]: best test :)

Comment (eleftherioszisis) [reply tests/test_morphology_readers.cpp:161]: brilliant!

File: tests/test_mitochondria.cpp [hunk 2]
```diff
@@ -8,72 +8,65 @@
...
 
 TEST_CASE("mitochondria", "[mitochondria]") {
-    const auto mito = morphio::Morphology("data/h5/v1/mitochondria.h5").mitochondria();
-
+    morphio::Morphology morph = morphio::Morphology("data/h5/v1/mitochondria.h5");
+    morphio::Mitochondria mito = morph.mitochondria();
     REQUIRE(mito.rootSections().size() == 2);
 
-    auto rootSection = mito.rootSections()[0];
+    morphio::MitoSection rootSection = mito.rootSections().at(0);
```
Comment (eleftherioszisis) [inline tests/test_mitochondria.cpp:20]: Doesn't gsl complain `at` here?

**Review by eleftherioszisis — APPROVED** [review]

File: tests/test_mitochondria.cpp (hunk 2 above)
Comment (mgeplf) [reply tests/test_mitochondria.cpp:20]: `mito.rootSections()` returns a vector here

**Pull request merged**